addOns/reports/reports.gradle.kts

```diff
@@ -15,6 +15,18 @@ zapAddOn {
         author.set("ZAP Dev Team")
         url.set("https://www.zaproxy.org/docs/desktop/addons/report-generation/")
         extensions {
+            register("org.zaproxy.addon.reports.ExtensionReports") {
+                classnames {
+                    restricted.set(listOf("org.zaproxy.addon.reports.automation"))
+                }
+                dependencies {
+                    addOns {
+                        register("automation") {
```

Comment:
This means that to create reports you need the automation add-on.

Reply:
Thats correct. The plan is to include both as standard for the next release.

Reply:
Is this temporary? Ideally it should not need to depend.